pywatts/core/pipeline.py

```diff
@@ -219,33 +219,31 @@ def _run(self, data: Union[pd.DataFrame, xr.Dataset], mode: ComputationMode, sum
 
         if isinstance(data, pd.DataFrame):
             data = data.to_xarray()
-        elif isinstance(data, dict):
+            data = {key: data[key] for key in data.data_vars}
+        elif isinstance(data, xr.Dataset):
+            data = {key: data[key] for key in data.data_vars}
+
+        if isinstance(data,dict):
```

Comment:
We could use 'elif' here as above? Otherwise it checks the types again because we convert it to DataFrames and Datasets to Dicts. Or is that wanted?